crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -1112,6 +1224,56 @@ module deepbook::clob_v2 {
         clock: &Clock,
         ctx: &mut TxContext,
     ): (Coin<BaseAsset>, Coin<QuoteAsset>) {
+        let (base_coin, quote_coin, _metadata) = place_market_order_int(
```

Comment:
I suppose the fact that the gas charges for this function are going to change is not a big deal right?
I understand that they can always very depending on a lot of factors but asking in case....

Reply:
this is addressed in the next commit I noticed. Thanks